release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -273,6 +273,20 @@ Deployment now supports xref:../installing/installing_bare_metal_ipi/ipi-install
 
 Deployment now performs introspection on nodes to ensure that nodes meet installation requirements instead of generating errors if they do not.
 
+[id="ocp-4-6-installation-osp-availability-zones"]
+==== {rh-openstack} availability zones selection at installation
+
+You can now select {rh-openstack} Compute (Nova) availability zones while installing a cluster on {rh-openstack}.
```

Comment:
Perhaps for the first mention in this description, we can use `{rh-openstack-first}` attribute, just to help out those Ctrl+f'ing for "OpenStack".

Reply:
It is done.